digitalocean/resource_digitalocean_kubernetes_node_pool.go

```diff
@@ -387,6 +244,11 @@ func digitaloceanKubernetesNodePoolUpdate(client *godo.Client, pool map[string]i
 		req.Labels = expandLabels(pool["labels"].(map[string]interface{}))
 	}
 
+	if pool["taint"] != nil {
```

Comment:
This will always be nil here. This is called from `resourceDigitalOceanKubernetesNodePoolUpdate`. The taints must also be set there. 

https://github.com/digitalocean/terraform-provider-digitalocean/blob/9ee1da849a561a4619ddacd37bd717e12058b303/digitalocean/resource_digitalocean_kubernetes_node_pool.go#L93-L110